pkg/operator/volumesnapshotclasscontroller/volumesnapshotclasscontroller.go

```diff
@@ -59,6 +64,13 @@ func (c *VolumeSnapshotClassController) sync(ctx context.Context, syncCtx factor
 		return nil
 	}
 
+	crdName := "volumesnapshotclasses.snapshot.storage.k8s.io"
+	_, err = c.apiExtClient.ApiextensionsV1().CustomResourceDefinitions().Get(ctx, crdName, metav1.GetOptions{})
```

Comment:
@bertinatto maybe I should have noticed it earlier, do our operators have GET permissions on CRD? They will silently stop deploying snapshot classes if not.

Reply:
@jsafrane yes: https://github.com/openshift/cluster-storage-operator/pull/323/files

Reply:
https://github.com/openshift/cluster-storage-operator/pull/323